crates/ark/src/lsp/completions/completion_item.rs

```diff
@@ -416,14 +425,14 @@ pub(super) unsafe fn completion_item_from_lazydata(
     let promise_strategy = PromiseStrategy::Simple;
 
     // Lazydata objects are never functions, so this doesn't really matter
-    let parameter_hints = ParameterHints::Enabled;
+    let parameter_hints = ParameterHints::Disabled;
 
     match completion_item_from_symbol(name, env, Some(package), promise_strategy, &parameter_hints)
     {
-        Some(item) => item,
-        None => {
+        Ok(item) => return Ok(item),
```

Comment:
```suggestion
        Ok(item) => Ok(item),
```

When the match is the last statement we prefer fallthrough